examples/service-clients/azure-client/external-controller/package.json

```diff
@@ -11,13 +11,26 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
-	"main": "dist/index.js",
-	"module": "lib/index.mjs",
-	"types": "dist/index.d.ts",
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/src/app.d.ts",
+				"default": "./lib/src/app.js"
+			},
+			"require": {
+				"types": "./dist/src/app.d.ts",
+				"default": "./dist/src/app.js"
+			}
+		}
+	},
```

Comment:
Note that main/module/types originally pointed to non-existent files.

I guess it didn't matter as long as webpack's config could find them.